src/applications/vaos/new-appointment/components/TypeOfCarePage/index.jsx

```diff
@@ -70,10 +70,6 @@ function TypeOfCarePage({
     }
   }, []);
 
-  if (!schema) {
```

Comment:
This was breaking the focus on the h1 when this page loaded, because the h1 wasn't rendered on the initial render